src/d.ts

```diff
@@ -36,20 +37,20 @@ export function w<S extends WidgetState, W extends Widget<S>, O extends WidgetOp
 	};
 }
 
-export function v(tag: string, options: VNodeProperties, children?: DNode[]): HNode;
+export function v(tag: string, properties: VNodeProperties, children?: DNode[]): HNode;
 export function v(tag: string, children: DNode[]): HNode;
 export function v(tag: string): HNode;
-export function v(tag: string, optionsOrChildren: VNodeProperties = {}, children: DNode[] = []): HNode {
+export function v(tag: string, propertiesOrChildren: VNodeProperties = {}, children: DNode[] = []): HNode {
 
-		if (Array.isArray(optionsOrChildren)) {
-			children = optionsOrChildren;
-			optionsOrChildren = {};
+		if (Array.isArray(propertiesOrChildren)) {
```

Comment:
I renamed the the property of `v` to `propertiesOrChildren` in line with the type `VNodeProperties`